app/src/main/java/com/jwisozk/flightreviews/ReviewAdapter.kt

```diff
@@ -0,0 +1,131 @@
+package com.jwisozk.flightreviews
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.CheckBox
+import android.widget.EditText
+import android.widget.RatingBar
+import android.widget.TextView
+import android.widget.Button
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.RecyclerView
+import com.jwisozk.flightreviews.util.ParamEnum
+
+class ReviewAdapter(
+    val viewModel: ParamViewModel
+) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
Let's extend `ListAdapter` here, this enables us to use lists of cells to specify items layout, like it is in guides and doesn't bound us to items position